browser-test/script.map

```diff
@@ -3,7 +3,7 @@
   "file": "script.js",
   "sourceRoot": "..",
   "sources": [
-    "browser-test/script.coffee"
+    "browser-test\\script.coffee"
```

Comment:
I didn't notice that. `script.map` is a build output, but I can revert the incidental change